src/Gemfile

```diff
@@ -13,6 +13,7 @@ gem 'sass-rails', '~> 6'
 gem 'webpacker', '~> 5.0'
 
 group :development, :test do
+  gem 'byebug'
   gem 'factory_bot_rails'
   gem 'pry-rails'
```

Comment:
I suggest you use `pry-byebug`  as it contains byebug and pry.
 https://github.com/deivid-rodriguez/pry-byebug

Reply:
why should we add byebug while we have pry?

Reply:
some people prefer `byebug`, so I suggest to use `pry-byebug` to avoid 
adding to gems for the same purpose.

Reply:
that's not a good enough reason to add another dependency, I'll have to request removing it from the changes